doc/devel/contributing.rst

```diff
@@ -37,8 +37,15 @@ particularly suited for new contributors by the label
 These are well documented issues, that do not require a deep understanding of
 the internals of Matplotlib. The issues may additionally be tagged with a
 difficulty. ``Difficulty: Easy`` is suited for people with little Python experience.
-``Difficulty: Medium`` and ``Difficulty: Hard`` are not trivial to solve and
-require more thought and programming experience.
+``Difficulty: Medium`` and ``Difficulty: Hard`` require more programming experience.
+This could be for a variety of reasons, among them:
```

Comment:
Needs a blank line between paragraph and the list.

Reply:
I'd prefer a more active tense (not sure if this is the correct term), so fewer *could, would* and more *can, are*.

(Also *might* often sounds a bit more "unrealistic" than *may*. *I may go shopping today. vs I might go to Mars.*)

(These are all things I tend to do myself, but trying hard to avoid. It is somehow more natural to not be that direct, but often there is no harm. Rather the opposite.)

Reply:
I think they are called modal auxiliary verbs in English?!